### DISCUSSION TITLE
Add feature - lerna list

### DISCUSSION THREAD

**Discussion by johnhomantaring** [ghiscoding/lerna-lite#170] [Q&A] [answered]
Are you planning to migrate lerna list or any alternative like using the lerna info? Need the list to identify the package path to validate that there are no typos in the bundle names of packages as declared in package.json.

**Comment by ghiscoding** [comment, marked as answer]
I wasn't planning on adding any more commands, well anyway I said that a month ago and still decided to add 2 more small commands `init` and `info` in default CLI. I'm not saying no, but it would certainly have to be an optional command, it has extra lerna packages that are not currently in Lerna-Lite like this [listable](https://github.com/lerna/lerna/tree/main/utils/listable) which also requires extra external libs. In short I don't want to make Lerna-Lite cli & core bigger than they are now, but I wouldn't mind as an optional command. 

The reason I don't want to grow the core code any bigger is that from what I can see, a big portion of Lerna-Lite's users are only interested in `version` and `publish` commands and there's a very small portion using `run` and a micro portion using `exec` hence a good decision to make them optional downloads. A lot of users will want to use their other package manager or lib to do some of the old Lerna's command like `run` with `pnpm run` or with Turborepo or whatever else. 

Long story short, I wouldn't mind adding it as an optional command, would you like to look into making a PR for that?
A good starting point could be to look at how I added `exec` in PR #135

**Reply by ghiscoding** [reply]
@johnhomantaring hey so I looked at adding the `list` command as an optional command in PR #171, it will be an optional command. I will only merge it once the other opened PR is merge first.

**Reply by johnhomantaring** [reply]
thank you very much
